src/python/CRABClient/Commands/getlog.py

```diff
@@ -107,3 +85,31 @@ def validateOptions(self):
                 ex = MissingOptionException(msg)
                 ex.missingOption = "jobids"
                 raise ex
+
+
+    def retrieveShortLogs(self, webdir, proxyfilename):
+        self.logger.info("Retrieving...")
+        success = []
+        failed = []
+        for item in self.options.jobids:
+            jobid = str(item[1])
+            filename = 'job_out.' + jobid + '.0.txt'
+            url = webdir + '/' + filename
+            try:
+                getFileFromURL(url, self.dest + '/' + filename, proxyfilename)
+                self.logger.info('Retrieved %s' % (filename))
+                success.append(filename)
+                retry = 1 #To retrieve retried joblog, if there is any.
+                while urllib.urlopen(webdir + '/' + 'job_out.' + jobid + '.' + str(retry) + '.txt').getcode() == 200:
```

Comment:
This doesn't work with proxied webdir right?
And I checked that
```
urllib.URLopener(key_file = proxyfilename, cert_file = proxyfilename).open(url)
```
raises IOError with http_error 404 when I use a url with a filename that doesn't exist:
```
>>> urllib.URLopener(key_file = proxyfilename, cert_file = proxyfilename).open(url)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/external/python/2.6.8-comp9/lib/python2.6/urllib.py", line 207, in open
    return getattr(self, name)(url)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/external/python/2.6.8-comp9/lib/python2.6/urllib.py", line 457, in open_https
    return self.http_error(url, fp, errcode, errmsg, headers)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/external/python/2.6.8-comp9/lib/python2.6/urllib.py", line 379, in http_error
    return self.http_error_default(url, fp, errcode, errmsg, headers)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/external/python/2.6.8-comp9/lib/python2.6/urllib.py", line 385, in http_error_default
    raise IOError, ('http error', errcode, errmsg, headers)
IOError: ('http error', 404, 'Not Found', <httplib.HTTPMessage instance at 0x7f0dacee5170>)
```